appserver/web/web-glue/src/main/java/com/sun/enterprise/web/session/SessionCookieConfig.java

```diff
@@ -282,6 +293,10 @@ public String toString() {
             sb.append(", domain=");
             sb.append(_domain);
         }
+        if (sameSite != null) {
+            sb.append(", sameSite=");
```

Comment:
here also split more the append calls and use sameSite literal value on a constant